packages/vats/src/bootstrap.js

```diff
@@ -202,10 +241,12 @@ export function buildRootObject(vatPowers, vatParameters) {
     ) || { amount: '0' };
 
     // Now we can bootstrap the economy!
-    const bootstrapPaymentValue = Nat(BigInt(centralBootstrapSupply.amount));
+    const bankBootstrapSupply = Nat(BigInt(centralBootstrapSupply.amount));
+    // Ask the treasury for enough RUN to fund both AMM and bank.
+    const bootstrapPaymentValue = bankBootstrapSupply + ammDepositValue;
     // NOTE: no use of the voteCreator. We'll need it to initiate votes on
     // changing Treasury parameters.
-    const { treasuryCreator, _voteCreator } = await installEconomy(
+    const { treasuryCreator, _voteCreator, ammFacets } = await installEconomy(
```

Comment:
This voteCreator NOTE looks like a TODO; is it covered by an issue? #3924 looks close enough.